tools/bbc/bbc.cpp

```diff
@@ -147,7 +148,14 @@ void convertFortranSourceToMLIR(
 
   if (EmitLLVM) {
     pm.addPass(fir::createFIRToLLVMPass(nameUniquer));
-    pm.addPass(fir::createLLVMDialectToLLVMPass(OutputFilename + ".ll"));
+    std::error_code ec;
+    llvm::ToolOutputFile out(OutputFilename + ".ll", ec,
+                             llvm::sys::fs::OF_None);
```

Comment:
Is this change required for the tests or is this just a clean-up to align with how LLVM creates output file ?

Reply:
It's a personal requirement since I want tools that can write to stdout to make it easier to build a pipeline for lit tests. :)